doc/sphinx/source/releases.rst

```diff
@@ -19,11 +19,19 @@ The code is tagged to contextualize the versioning, mark significant
 developments and to mark versions used to produce main results. The
 significant releases since the code was made public are:
 
+`Version 4.0.10 <https://github.com/NNPDF/nnpdf/releases/tag/4.0.10>`_
+    Last release of the 4.0 era. Runcards for 4.0 work in this release will work
+    but are no longer used the old tools, commondata or theories.
```

Comment:
Runcards for 4.0 work in this release will work but are no longer used the old tools, commondata or theories.

This is not an proper sentence